sell-loot.lic

```diff
@@ -60,23 +78,6 @@ class SellLoot
     end
   end
 
-  def parse(setting)
-    arg_definitions = [
-      [
-        { name: 'amount', regex: /\d+/i, variable: true, description: 'Number of coins to keep' },
-        { name: 'type', regex: /\w+/i, variable: true, description: 'Type of coins to keep' }
-      ],
-      []
-    ]
-
-    args = parse_args(arg_definitions)
-
-    keep_amount = args.amount || setting[0] || 3
-    keep_type = args.type || setting[1] || 'silver'
-
-    convert_to_copper(keep_amount, keep_type)
-  end
```

Comment:
Parsing of arguments and determining how much coin to keep on your person is now part of the `initialize` method